benchmark/Cpp/Savina/src/concurrency/Philosopher.lf

```diff
@@ -1,58 +1,20 @@
 /**
- * Concurrency benchmark from the Savina benchmark suite.
- * See https://shamsimam.github.io/papers/2014-agere-savina.pdf.
+ * Copyright (C) 2020 TU Dresden
  * 
  * This implementation stays close to the Savina implementation
```

Comment:
These comments could be improved.

Reply:
Agreed, this is still the old comment and I will update it.